playbooks/frontend.yml

```diff
@@ -104,35 +104,23 @@
         - ryanmacg
         - zahids
 
-    - name: Ensure /etc/ecs directory exists
-      file: path=/etc/ecs state=directory
-
-    - name: ECS Cluster Registration
```

Comment:
Why aren't you writing this file to out anymore?  Commit message doesn't mention it.

Reply:
Commit message does mention this.  Does it need to be more detailed?

Reply:
Oh this is the EC2 instance registration. 🤦‍♂️